### PR TITLE
[Term Entry] Python: NumPy built-in-functions Variance 

### LINKED ISSUE #4476: [Term Entry] Python:NumPy built-in-functions Variance
### Reason/inspiration (optional)

"We would like a new term entry on the `built-in-functions` term for [Numpy](https://www.codecademy.com/resources/docs/numpy): Variance. The entry should go in a new file under `docs/content/numpy/concepts/built-in-functions/terms/variance/variance.md`.

The entry should describe the term and how it is used within statistical analysis. Also include: 
- An "Example" section that explains readers how to use the variance statistical function.
- Should contain informations for functions and syntax.
- Should have a python codebyte with working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by luca1iu**
### Description

- Added the NumPy built-in-functions numpy.var() term
- Closes #4476 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - Structures
+  - Arrays
+  - Functions
+  - NumPy
+CatalogContent:
+  - learn-python-3
+  - paths/data-science
+---
+
+The `.var()` function compute the variance of the given data (array elements) along the specified axis(if any). This value represents the average of the squared differences from the mean and provides a measure of how spread out the values are in the array.
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:18]:
```suggestion
The `.var()` [function](https://www.codecademy.com/resources/docs/numpy/built-in-functions) computes the variance of the elements of an array along the specified axis (if any). This value represents the average of the squared differences from the mean and provides a measure of how spread out the elements are in the array.
```

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Syntax
+
+```pseudo
+numpy.var(array, axis=None)
+```
+
+Where `array` is the input array for which the variance needs to be calculated. `axis (optional)` specifies the axis along which the variance is calculated. If `axis` is not provided, the variance is calculated for the entire array. If `axis` is specified,then the variance is calculated along that axis. For a 2D array, if axis=0, the variance is calculated column-wise, and if axis=1, the variance is calculated row-wise.
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:26]:
```suggestion
- `array`: The array for which the variance is to be calculated.
- `axis`: An optional parameter that specifies the axis along which the variance is to be calculated.

If `axis` is not provided, the variance is calculated for the entire array. If `axis` is specified, then the variance is calculated along that axis. For a 2D array, if `axis=0`, the variance is calculated column-wise and if `axis=1`, the variance is calculated row-wise.
```

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Example
+
+The example below creates a 2D array and calculates the overall variance, column-wise variance, and row-wise variance.
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:30]:
```suggestion
The below example creates a 2D array and calculates the overall variance, column-wise variance, and row-wise variance of the array using the `.var()` function:
```

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+```py
+import numpy as np
+
+# Creating a 2D array
+arr = np.array([[1, 2, 3],[4, 5, 6],[7, 8, 9]])
+
+# the variance of the entire array
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:38]:
```suggestion
# Calculating the overall variance
```

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+variance_all = np.var(arr)
+print(variance_all)
+
+# the variance along axis=0 (column-wise)
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:42]:
```suggestion
# Calculating the column-wise variance (axis=0)
```

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+variance_axis_0 = np.var(arr, axis=0)
+print(variance_axis_0)
+
+# the variance along axis=1 (row-wise)
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:46]:
```suggestion
# Calculating the row-wise variance (axis=1)
```

**Review by Sriparno08 — COMMENTED** [review]
Hey @luca1iu, thanks for contributing to Codecademy!

I've listed all the modifications that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Good job, @luca1iu!

This looks good for a second review 👍

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+- `array`: The array for which the variance is to be calculated.
+- `axis`: An optional parameter that specifies the axis along which the variance is to be calculated.
+
+If `axis` is not provided, the variance is calculated for the entire array. If `axis` is specified, then the variance is calculated along that axis. For a 2D array, if `axis=0`, the variance is calculated column-wise and if `axis=1`, the variance is calculated row-wise.
+
+## Example
+
+The below example creates a 2D array and calculates the overall variance, column-wise variance, and row-wise variance of the array using the `.var()` function:
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:32]:
```suggestion
The following example creates a 2D array and calculates the overall variance (variance of the flattened array), column-wise variance, and row-wise variance of the array using the `.var()` function:
```

File: content/numpy/concepts/built-in-functions/terms/variance/variance.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+variance_all = np.var(arr)
+print(variance_all)
+
+# Calculating the column-wise variance (axis=0)
+variance_axis_0 = np.var(arr, axis=0)
+print(variance_axis_0)
+
+# Calculating the row-wise variance (axis=1)
+variance_axis_1 = np.var(arr, axis=1)
+print(variance_axis_1)
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/variance/variance.md:50]:
```suggestion
import numpy as np

# Creating a 2D array
array_2d = np.array([[1, 2, 3],[4, 5, 6],[7, 8, 9]])

# Calculating the overall variance
overall_variance = np.var(array_2d)
print(overall_variance)

# Calculating the column-wise variance (axis=0)
column_variance = np.var(array_2d, axis=0)
print(column_variance)

# Calculating the row-wise variance (axis=1)
row_variance = np.var(array_2d, axis=1)
print(row_variance)
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@luca1iu 
I've listed below some changes. Please make them as soon as possible.

**Review by letsbuildanything — APPROVED** [review]
LGTM! 🚀

**Pull request merged**